spring-cloud-stream-rxjava/src/main/java/org/springframework/cloud/stream/annotation/rxjava/SubjectMessageHandler.java

```diff
@@ -141,7 +141,6 @@ public int getPhase() {
 	}
 
 	@Override
-	//todo: support module input type
```

Comment:
Removing this as it will be addressed as part of https://github.com/spring-cloud/spring-cloud-stream/issues/458 in general